evergreen.yaml

```diff
@@ -193,55 +201,48 @@ tasks:
 #           Buildvariants             #
 #######################################
 buildvariants:
-  - name: race-detector
-    display_name: Race Detector
-    expansions:
-      GOROOT: /opt/golang/go1.16
-      RACE_DETECTOR: true
-      MONGODB_URL: https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-4.0.3.tgz
-    run_on:
-      - archlinux-new-large
-    tasks:
-      - ".test"
-
   - name: lint
-    display_name: Lint (Arch Linux)
+    display_name: Lint (Ubuntu 22.04)
```

Comment:
We can just call this `Lint` since the Linux distro isn't that important.

Reply:
Fine by me.